.github/workflows/documentation-update.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           path: builders-portal
           token: ${{ secrets.ARABOT_PAT }}
       - name: Copy docs
-        run: cp -R packages/contracts/docs/generated/* $GITHUB_WORKSPACE/builders-portal/docs/smart-contracts/
+        run: cp -R packages/contracts/docs/core/* $GITHUB_WORKSPACE/builders-portal/docs/core/
```

Comment:
don't forget to delete the folder `smart-contracts` in the builder-portal repo.
could you please add a `rm -rf` to the folder, so we can remove outdated files too.

Reply:
Fixed